README.md

```diff
@@ -122,6 +122,34 @@ TiAuth.initialize(function(token, user) {
 });
 ```
 
+### `loadGrants` *(callback)*
+
+This method allows you to load the grants information for the current user. See
+the API documentation of the TI API for further information about the endpoint.
```

Comment:
@trothe It’d be awesome if you could link the `API documentation of the TI API` portion of this sentence to any existing documentation of the `/api/v1/grants` endpoint that may exist in the Wiki. If there’s no such documentation available let me know and I’ll delete this whole sentence (because I don’t want to document the endpoint as part of our README either).